components/atlas-uvs/index.html

```diff
@@ -66,7 +66,17 @@ <h1><i>atlas-uvs</i> Component for <a href="https://aframe.io">A-Frame</a></h1>
     <h2>Examples</h2>
 
     <ul id="examples">
-      
+      <li>
+        <a href="examples/static/" style="background-color: #222; background-image: url()"></a>
+        <h3>Static with A-Frame 1.2</h3>
+        <p>Simple atlas example</p>
+      </li>
+      <li>
+        <a href="examples/dynamic/" style="background-color: #222; background-image: url()"></a>
+        <h3>Dynamic with A-Frame 0.8.2</h3>
```

Comment:
Dynamic is not possible with 1.2.0?

Reply:
I didn't want to touch it since it wasn't doucmented but just tested and it does work with new version, so I will update

Reply:
Updated and confirmed working